docs/scripts_fsharp/ArcInvestigation.fsx

```diff
@@ -20,7 +20,7 @@ open FsSpreadsheet.Net
 
 let fswb = ArcInvestigation.toFsWorkbook investigation_comments
 
-fswb.ToFile("test2.isa.investigation.xlsx")
+fswb.ToXlsxFile("test2.isa.investigation.xlsx")
```

Comment:
is this updated for python and javascript alike?

Reply:
Yup